app/routes/audit.js

```diff
@@ -0,0 +1,19 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class AuditRoute extends Route {
+  @service store;
+  @service('requests/audit/get-audit-logs-service') requestAuditService;
+
+  async model() {
+    // Fetch the audit logs from the API
+    await this.requestAuditService.getAuditLogs();
```

Comment:
Logs returned from the service are `this.store.peekAll('audit-log')` so you don't have to do it later on.

Reply:
On this - i had tried removing the peekAll from the route but found it broke template rendering. After examining what I was getting from my handler via the service call I noticed it was not a proxy array which is what the template expects I believe. So adding another peekAll in the route gave me that proxy Array to pass to the template which worked.

`Route data from service:  {request: {…}, response: {…}, content: undefined, Symbol(DOC): true}
audit.js:27 Route PeekAll Data Proxy(Array) {0: {…}, 1: {…}, 2: {…}, 3: {…}, 4: {…}}`

So to me - I feel that perhaps I have too many peekAlls - but that I should remove the one from the handler as essentially it does not serve any purpose and the handlers function should only be to receive, tidy and push data into the store which its doing - so I have removed that peekAll from the service

And that leaves my route to peekAll and provide the data to the template - am I right in this? I may also remove them from the other handlers also as they serve no purpose I feel.

RE: https://github.com/Mr-Smyth/task-manager/pull/25/commits/508c55a7fd01caa91b1b8391215735b3aba862f4

Reply:
Some Interesting internals... Yeah, let's simplify it as you proposed.